ledger/participant-state/kvutils/app/src/main/scala/com/daml/ledger/participant/state/kvutils/app/ParticipantMode.scala

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.participant.state.kvutils.app
+
+sealed abstract class ParticipantMode
+
+object ParticipantMode {
```

Comment:
In line with the above recommendation on the key for the `participant` parameter you may want to call this `ParticipantRunMode` or `ParticipantServerMode` instead.